install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml

```diff
@@ -38,7 +38,7 @@ spec:
 {{- end }}
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          image: "{{ .Values.hub }}/grafana:{{ .Values.tag }}"
```

Comment:
is this correct that / after .values.hub?

Reply:
@linsun yes. See https://github.com/istio/istio/blob/918ee98fbfcb478fb3d8a5300e6177353733b4cc/install/kubernetes/helm/istio/charts/prometheus/templates/deployment.yaml#L33.

Reply:
ok thanks!

Reply:
@linsun if we decide to use the old syntax, then this wouldn't work. we'd have to rollback to the previous syntax (which is fine w/ me).